### PR TITLE
Add validations for maxItems and minItems

### PR DESCRIPTION
**Pull request by bsonntag**
This PR updates the `validate` utility to add support for the `maxItems` and `minItems` keywords.

### CODE REVIEW

File: src/utils/validate.js [hunk 1]
```diff
@@ -40,9 +40,11 @@ function getPropertyName(error: Object): string {
 
 function getErrorArgs(error) {
   switch (error.keyword) {
+    case 'maxItems':
```
Comment (franciscomorais) [inline src/utils/validate.js]: WDYT about add `maxProperties`/`minProperties` to objects and maybe `exclusiveMaximum`/`exclusiveMinimum` to keywords or numbers? Check this links below: 

https://ajv.js.org/keywords.html#maxproperties--minproperties
https://ajv.js.org/keywords.html#maximum--minimum-and-exclusivemaximum--exclusiveminimum

Comment (bsonntag) [reply src/utils/validate.js]: Sure, but I'll do that in a different PR.

Comment (bsonntag) [reply src/utils/validate.js]: Done in #16 and #17

Comment (bsonntag) [reply src/utils/validate.js]: Done in #16 and #17

**Pull request merged**